src/applications/financial-status-report/components/PreSubmitSignature.jsx

```diff
@@ -139,6 +139,19 @@ const PreSubmitSignature = ({
           <li>My bankruptcy history</li>
         </ul>
 
+        <p>
+          I have read and accept the
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            className="vads-u-margin-left--0p5"
+            href="https://www.va.gov/privacy-policy/"
```

Comment:
Might be better to use something like `href=${environment.BASE_URL}/privacy-policy` instead of hard coding va.gov